internal/cmd/iam/command_groupmapping_update.go

```diff
@@ -22,7 +20,7 @@ func (c *groupMappingCommand) newUpdateCommand() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: `Update the description of group mapping "pool-123456".`,
-				Code: `confluent iam group-mapping update pool-123456 --description "updated description"`,
+				Code: "confluent iam group-mapping update pool-123456 --description updated-description",
```

Comment:
Sorry for being ambiguous earlier. Our goal is to replicate what users would most likely want to input here. I think the majority of the time, users want to have spaces in their description. Any flag value or argument (resource) should ideally be lower-kebab-case since that's what's easiest to type.
```suggestion
				Code: `confluent iam group-mapping update pool-123456 --description "updated description"`,
```

Reply:
All good, just gonna revert the last commit then.